giskard-server/src/main/java/ai/giskard/web/rest/errors/EntityAccessControlException.java

```diff
@@ -0,0 +1,24 @@
+package ai.giskard.web.rest.errors;
+
+import lombok.Getter;
+
+public class EntityAccessControlException extends BadRequestAlertException {
+
+    public enum Entity {
```

Comment:
This enum can be moved outside and be reused, otherwise there are already 4 similar enum declarations